internal/server/auth/method/token/server.go

```diff
@@ -40,17 +41,25 @@ func (s *Server) RegisterGRPC(server *grpc.Server) {
 // CreateToken adapts and delegates the token request to the backing AuthenticationStore.
 //
 // Implicitly, the Authentication created will be of type auth.Method_TOKEN.
-// Name and Description are both stored in Authentication.Metadata.
+// Name, Description, and NamespaceKey are both stored in Authentication.Metadata.
 // Given the token is created successfully, the generate clientToken string is returned.
 // Along with the created Authentication, which includes it's identifier and associated timestamps.
 func (s *Server) CreateToken(ctx context.Context, req *auth.CreateTokenRequest) (*auth.CreateTokenResponse, error) {
+	metadata := map[string]string{
+		storageMetadataNameKey: req.GetName(),
+	}
+
+	if req.GetDescription() != "" {
+		metadata[storageMetadataDescriptionKey] = req.GetDescription()
+	}
+	if req.GetNamespaceKey() != "" {
+		metadata[storageMetadataNamespaceKey] = req.GetNamespaceKey()
+	}
```

Comment:
From the UI perspective it looks like a namespace is always going to be present, but someone could just create a Token without namespace from the API perspective right?

Reply:
Namespace doesn't always have to be present. Can be none if they don't click the checkbox and pick one. I prob should add a demo gif as well. Namespace is optional in the ui for token creation